services/users/handler.js

```diff
@@ -338,7 +331,9 @@ export const favouriteEvent = async (event, ctx, callback) => {
 
     let expressionAttributeValues;
     expressionAttributeValues = {
-      ":eventsIDAndYear": docClient.createSet([eventIDAndYear]) // set data type, for updateExpression
+      ":eventsIDAndYear": marshall({
```

Comment:
I'm not familiar with marshalling, but just wanted to check if we need it here? Our docClient uses lib-dynamodb, which is supposed to automatically perform it https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-util-dynamodb/

Reply:
Yeah looked into it. I think either works, but createSet is preferred for dynamic languages. 

<img width="836" alt="Screenshot 2024-08-12 at 12 20 57 AM" src="https://github.com/user-attachments/assets/ab08e982-cbc4-47fb-a145-48c2234b3dd4">